scripts/runfit.jl

```diff
@@ -0,0 +1,92 @@
+working_dir = "/home/vekondra/ACEpotentials.jl/scripts"
```

Comment:
Something like this can be achieved by `@__DIR__()`; in general a personal path should never be in a public repository.